### DISCUSSION TITLE
What's your current setup for exporting traces using candlelight

### DISCUSSION THREAD

**Discussion by joe94** [xmidt-org/xmidt#53] [Q&A] [answered]
https://github.com/xmidt-org/candlelight currently supports exporting spans to `zipkin` and `jaeger`. We're looking into adding `otlp` support for exporting spans to Datadog. 

We're curious to hear about your current choice to export traces and how the experience has been using `candlelight`

**Comment by joe94** [comment]
@Sachin4403 ☝🏼

**Comment by Sachin4403** [comment, marked as answer]
Hi @joe94, Currently we are using`stdout` with `skipTraceExport=true`.  In future  we might use `jaegar`
